### PR TITLE
Run sql experiments_v1 alongside databricks

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

File: dags/main_summary.py [hunk 1]
```diff
@@ -219,6 +223,59 @@
...
+    subdag=export_to_parquet(
+        table="moz-fx-data-shared-prod:telemetry_derived.experiments_v1${{ds_nodash}}",
+        destination_table="sql_experiments_v1",  # TODO remove this line when removing the databricks job
+        arguments=[
+            "--partition-by",
+            "experiment_id",
+            "submission_date_s3",
+            "--select",
+            "*",
+            "'{{ds_nodash}}' AS submission_date_s3",
```
Comment (sunahsuh) [inline dags/main_summary.py]: Should this have a `--replace` flag before this line?

File: dags/main_summary.py [hunk 2]
```diff
@@ -219,6 +223,59 @@
...
+            "--replace",
+            "'{{ds_nodash}}' AS submission_date",
+            "SAFE_CAST(sample_id AS INT64) AS sample_id",
```
Comment (sunahsuh) [inline dags/main_summary.py]: Same here re: `--replace`?

Comment (relud) [reply dags/main_summary.py]: `--replace` uses `nargs="+"`, so we specify the flag once, and 1 or more values

File: dags/main_summary.py (hunk 1 above)
Comment (relud) [reply dags/main_summary.py]: this is an arg for `--select`, which also uses `nargs="+"`

**Pull request merged**